itez/beneficiary/admin.py

```diff
@@ -6,7 +6,11 @@
     WorkDetail,
     AgentDetail,
     Beneficiary,
-    BeneficiaryParent
+    BeneficiaryParent,
+    Facility,
+    FacilityType,
+    ImplementingPartner,
+    ServiceProvider
```

Comment:
`ServiceProviderPersonnel` is what I said we call this model and not `ServiceProvider`

Reply:
Didn't get that part. Let me fix it.